src/applications/personalization/profile/components/military-information/MilitaryInformation.jsx

```diff
@@ -208,16 +204,10 @@ const MilitaryInformation = ({ militaryInformation, veteranStatus }) => {
   return (
     <div>
       <Headline>Military information</Headline>
-      <DowntimeNotification
-        appTitle="Military Information"
-        render={handleDowntimeForSection('military service')}
-        dependencies={[externalServices.emis]}
-      >
-        <MilitaryInformationContent
-          militaryInformation={militaryInformation}
-          veteranStatus={veteranStatus}
-        />
-      </DowntimeNotification>
```

Comment:
After dipping into the BE for this endpoint it looks like we are using VA Profile for the military information. Can you update this to be `externalServices.vaProfile` instead of yanking it out please?

Reply:
Absolutely, added `externalServices.vaProfile` as the dependency.